### PR TITLE
Fix #1: Add support for Windows

### LINKED ISSUE #1: Add support for Windows

### PR DESCRIPTION
**Pull request by lmorchard**
* Change findPorts() to support multiple commands for win32

* Add win32 command output parser and test cases

* Small refactoring of tests

### CODE REVIEW

**Comment by lmorchard** [comment]
Seems like it works on my Win8 machine. Started up a simulator and then this:

C:\Users\lmorchard\Documents\devel\node-firefox-find-ports [3-windows +1 ~0 -0 !]> node -e "require('./index')().then(console.log)"
[ { type: 'b2g', port: 56456, pid: 8060 } ]

**Comment by lmorchard** [comment]
Well, my home machine is 64-bit, but I think node just calls the platform "win32" no matter what

**Comment by tofumatt** [comment]
Cool then, r+ :+1:

File: lib/parsers.js [hunk 1]
```diff
@@ -49,9 +49,67 @@ function parserLinux(lines, search) {
...
+  var netstatLines = lines[1];
+  if (!isNonEmptyArray(netstatLines)) {
+    return [];
+  }
+
+  // Example tasklist syntax:
+  // b2g-bin.exe                  16180 RDP-Tcp#2                  1    129,560 K
+  var pidMap = {};
+  var tasklistRegex = new RegExp(
+    '^(' + search.join('|') + ')(?:-bin)?\\.exe\\s+([\\d]+)'
```
Comment (lmorchard) [inline lib/parsers.js]: Yeah, though the line was >90 col wide, so I thought that might be a bit more readable. I can join it up though

Comment (tofumatt) [inline lib/parsers.js]: Nah, > 90 is too long. I checked and your call was right :-)

**Pull request merged**